python/pynessie/client/nessie_client.py

```diff
@@ -148,18 +148,18 @@ def list_keys(
         hash_on_ref: str = None,
         max_result_hint: Optional[int] = None,
         page_token: Optional[str] = None,
-        query_expression: Optional[str] = None,
+        query_filter: Optional[str] = None,
```

Comment:
the only reason to name this `query_filter` and not `filter` is to not shadow the built-in `filter`